build.gradle

```diff
@@ -1,4 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
+buildscript {
+    repositories {
+        google()
+    }
+    dependencies {
+        var nav_version = "2.5.1"
+        classpath("androidx.navigation:navigation-safe-args-gradle-plugin:$nav_version")
```

Comment:
This is the plugin I mentioned in the previous PR
Add it there too and make the changes to use Directions classes and their methods everywhere